### DISCUSSION TITLE
Customize Onboarding PR

### DISCUSSION THREAD

**Discussion by ceng-p** [renovatebot/renovate#28496] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Running on Azure Devops v37.288.0

### Please tell us more about your question or problem

Hello. I saw there's an option to configure pull request contents, in general for dependencies, but I was looking for a way to customize the onboarding PRs themselves, is this something that's supported?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Some parts are customizable, you'll need to be more specific about what you want

**Reply by ceng-p** [reply]
@rarkins I'm in azure devops; I want to modify the contents of the pull request descriptions with context for our organization. We also have presets available that I want to introduce in the Onboarding PR so repository maintainers get the information they need.

**Reply by rarkins** [reply, marked as answer]
You can definitely choose which presets are included by default. Onboarding PR body is not customizable currently. Again, it would help if you can be more specific

**Reply by ceng-p** [reply]
If it's not customizable currently that answers my question 🥲
